mappings/net/minecraft/client/multiplayer/report/ChatAbuseReport.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/unmapped/C_nfpruzew net/minecraft/client/multiplayer/report/ChatAbuseReport
+	FIELD f_piqdcsrs selectedMessages Lit/unimi/dsi/fastutil/ints/IntSet;
+	METHOD m_zjhlvpic toggleMessageSelected (ILcom/mojang/authlib/minecraft/report/AbuseReportLimits;)V
+		ARG 1 message
```

Comment:
Not a fan of the word "message" duplication here, or of calling an `int` "message".
How about either
```suggestion
	METHOD m_zjhlvpic toggleSelected (ILcom/mojang/authlib/minecraft/report/AbuseReportLimits;)V
		ARG 1 messageIndex
```
or
```suggestion
	METHOD m_zjhlvpic toggleMessageSelected (ILcom/mojang/authlib/minecraft/report/AbuseReportLimits;)V
		ARG 1 index
```